docs/dg/dev/troubleshooting/troubleshooting-performance-issues/redis-session-lock-excpetions.md

```diff
@@ -67,3 +99,46 @@ Explore [architecture performance guidelines](/docs/dg/dev/guidelines/performanc
 
 ### Leverage APIs
 Headless scenarios aren't usually impacted by Session Locking challenges. Evaluate if you can adjust calls that would normally target Yves to target APIs instead.
+
+### Not using locks
+
+```php
+// config/Shared/config_default.php
+$config[SessionConstants::YVES_SESSION_SAVE_HANDLER] = SessionRedisConfig::SESSION_HANDLER_REDIS;
+```
+This approach optimistically assumes that there will be no concurrent requests from the client, which could otherwise cause session data inconsistencies. 
+For instance, this could occur if products are added to the cart simultaneously from multiple browser tabs.
+
+{% info_block warningBox "Disabling session locking" %}
+
+Disabling session locking is possible but should only be done after thoroughly assessing the risks and implementing necessary safeguards.
+For optimal performance, data integrity, and customer experience, we strongly recommend keeping session locks enabled and focusing on optimizing lock management and application performance.
+
+{% endinfo_block %}
+
+### Lock Time-to-Live Configuration
+
+By default, the lock time-to-live (TTL) is set to `0` in the configuration, which means it will default to the PHP configuration value for `max_execution_time`, if it is set.
+```php
+// config/Shared/config_default.php
+$config[SessionRedisConstants::LOCKING_LOCK_TTL_MILLISECONDS] = 0;
+```
+This configuration is based on the assumption that all PHP processes will terminate correctly, ensuring the lock is released as part of the process termination.
+
+If there is a risk of PHP processes hanging or terminating unexpectedly, it is recommended to set the lock TTL in Redis to a value significantly lower than the maximum PHP execution time.
```

Comment:
I'd suggest removing the whole line: we're suggesting something not based on any data - "unexpectedly", "significantly lower"

Reply:
refactored